charts/agent-control/values.yaml

```diff
@@ -65,8 +87,8 @@ agent-control-cd:
   installer:
     image:
       registry:
-      repository: alpine/helm
-      tag: "3.17.1"
+      repository: newrelic/newrelic-agent-control-cli
```

Comment:
why do we have this and at the same time installationJob.agentControlCd?

Is it still WIP?

Reply:
It's still used on a couple places in `install-flux.yaml`. I think merging the two image configurations, under e.g. `toolkitImage`, might not be desirable, at least until we merge the two installation jobs in [NR-448615](https://new-relic.atlassian.net/browse/NR-448615).

I can take that task next and get rid of this. WDYT?

Reply:
Created task to clean up the values: https://new-relic.atlassian.net/browse/NR-450510